e2e-tests/batch-mode-tests.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+print_prefix="E2E: BATCH MODE TEST:"
+echo "${print_prefix} START OF BATCH MODE TEST"
+echo "${print_prefix} STARTING OPENMRS SERVER"
+docker-compose -f openmrs-compose.yaml up -d --remove-orphans
+openmrs_start_wait_time=0
+contenttype=$(curl -o /dev/null --head -w "%{content_type}\n" -X GET -u admin:Admin123 --connect-timeout 5 --max-time 20 http://localhost:8099/openmrs/ws/rest/v1/visittype 2>/dev/null | cut -d ";" -f 1)
+until [[ ${contenttype} == "application/json" ]]; do
+      sleep 60s;
+      echo "${print_prefix} WAITING FOR OPENMRS SERVER TO START"
+      contenttype=$(curl -o /dev/null --head -w "%{content_type}\n" -X GET -u admin:Admin123 --connect-timeout 5 --max-time 20 http://localhost:8099/openmrs/ws/rest/v1/visittype 2>/dev/null | cut -d ";" -f 1)
+      ((openmrs_start_wait_time+=1))
+      if [[ ${openmrs_start_wait_time} == 20 ]]; then
+        echo "${print_prefix} TERMINATING TEST AS OPENMRS TOOK TOO LONG TO START"
+        exit 1
+      fi
+done
+echo "${print_prefix} OPENMRS SERVER STARTED SUCCESSFULLY"
+echo "${print_prefix} BUILDING THE ANALYTICS PROJECT"
+echo "mvn clean install -DskipTests=true"
+mvn clean install -DskipTests=true
+test_dir=$(mktemp -d --tmpdir -t analytics_tests__XXXXXX)
+echo "${print_prefix} PARQUET FILES WILL BE WRITTEN INTO $test_dir DIRECTORY"
+echo "${print_prefix} RUNNING BATCH MODE WITH PARQUET SYNC"
+echo "java -cp $(echo batch/target/fhir-batch-etl-bundled-*.jar | cut -d " " -f 1) org.openmrs.analytics.FhirEtl --openmrsServerUrl=http://localhost:8099/openmrs --openmrsUserName=admin --openmrsPassword=Admin123  --searchList=Patient,Encounter,Observation --batchSize=20 --targetParallelism=20 --fileParquetPath='${test_dir}'/"
```

Comment:
nit: If you want to keep this, please prefix it with `${print_prefix}`; if it was just for debug, please remove it.

Reply:
@bashir2 you had initially requested that we should also print the commands we're running. That's the reason I added this